yb-voyager/cmd/importSchema.go

```diff
@@ -192,7 +209,46 @@ func importSchema() {
 	importSchemaCompleteEvent := createImportSchemaCompletedEvent()
 	controlPlane.ImportSchemaCompleted(&importSchemaCompleteEvent)
 
-	callhome.PackAndSendPayload(exportDir)
+	return nil
+}
+
+func packAndSendImportSchemaPayload(status string, errMsg string) {
+	if !callhome.SendDiagnostics {
+		return
+	}
+	payload := createCallhomePayload()
+	payload.MigrationPhase = IMPORT_SCHEMA_PHASE
+	payload.Status = status
+	targetDBDetailsBytes, err := json.Marshal(targetDBDetails)
+	if err != nil {
+		log.Errorf("callhome: error in parsing sourcedb details: %v", err)
+	}
+	payload.TargetDBDetails = string(targetDBDetailsBytes)
+	var errorsList []string
+	for _, stmt := range finalFailedSqlStmts {
+		parts := strings.Split(stmt, "*/\n")
+		errorsList = append(errorsList, strings.Trim(parts[0], "/*\n"))
```

Comment:
have you tested this part, what about sql statements? I dont see the code for ignoring that.
Are they being ignored or considered as a part of error msg?

Reply:
Yes, Its only taking the error msg from each entry and not the SQL stmts.